stories/RadioGroup.story.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import { Formik, Field, Form } from 'formik';
+import Wrapper from './Wrapper';
+import { action } from '@storybook/addon-actions';
+
+import RadioGroup from '../src/RadioGroup';
+import {
+  FormControlLabel,
+  Radio,
+  CircularProgress,
+} from '../node_modules/@material-ui/core';
+import FormValues from './FormValues';
+
+export default () => (
+  <Wrapper title="Radio Group">
+    <Formik
+      initialValues={{}}
+      onSubmit={(values, { setSubmitting }) => {
+        setTimeout(() => {
+          setSubmitting(false);
+          action('submit')(values);
+        }, 2000);
+      }}
+      render={({ submitForm, values, isSubmitting }) => {
+        return (
+          <Form>
+            <Field
+              name="radio button"
+              label="Radio Group"
+              component={RadioGroup}
+            >
+              <FormControlLabel
+                value="painting"
+                control={<Radio />}
```

Comment:
nit: Can you add a `disabled={isSubmittiing}` so that this is consistent with the other forms?